config/jobs/kubernetes/sig-node/dra-canary.yaml

```diff
@@ -90,7 +90,7 @@ presubmits:
           features=( $(grep '"DRA' pkg/features/kube_features.go | sed 's/.*"\(.*\)"/\1/') )
           echo "Enabling DRA feature(s): ${features[*]}."
           # Those additional features are not in kind.yaml, but they can be added at the end.
-          kind create cluster --retain --config <(cat test/e2e/dra/kind.yaml; for feature in ${features}; do echo "  ${feature}: true"; done) --image dra/node:latest
+          kind create cluster --retain --config <(cat test/e2e/dra/kind.yaml; for feature in ${features[@]}; do echo "  ${feature}: true"; done) --image dra/node:latest
```

Comment:
I should have looked for more occurrences of this bug...

Let's fix this directly in all jobs. We know now that `feature in ${features[@]}` works, no need for being extra careful.

=> https://github.com/kubernetes/test-infra/pull/34244

/hold

Reply:
/close

Solved via that PR instead.